Source/Core/Core/Boot/Boot.cpp

```diff
@@ -594,18 +520,46 @@ bool CBoot::BootUp(std::unique_ptr<BootParameters> boot)
 
     bool operator()(const BootParameters::IPL& ipl) const
     {
-      NOTICE_LOG_FMT(BOOT, "Booting GC IPL: {}", ipl.path);
-      if (!File::Exists(ipl.path))
+      const std::string path = SConfig::GetInstance().m_strBootROM;
+      NOTICE_LOG_FMT(BOOT, "Booting GC IPL: {}", path);
+
+      ExpansionInterface::CEXIIPL* exi_ipl = static_cast<ExpansionInterface::CEXIIPL*>(
+          ExpansionInterface::GetChannel(0)->GetDevice(2));
+      DEBUG_ASSERT(exi_ipl->m_device_type == ExpansionInterface::EXIDeviceType::MaskROM);
+
+      if (!exi_ipl->IsROMLoaded())
       {
         if (ipl.disc)
           PanicAlertFmtT("Cannot start the game, because the GC IPL could not be found.");
         else
-          PanicAlertFmtT("Cannot find the GC IPL.");
+          PanicAlertFmtT("Could not find the GC IPL.");
         return false;
       }
 
-      if (!Load_BS2(ipl.path))
-        return false;
+      // MSR.IP = 1;
```

Comment:
Should this be further down with the other `MSR` initializations, and uncommented?

Reply:
According to yagcd, MSR.IP is set to 1 when the GameCube is powered on.  It's not part of the HLE hack that is being used, but I thought I wanted to document it somehow.  I think I'll remove this, now, since I want to try putting the IPL at physical memory 0xFFF00000 for a future PR.